x-pack/plugin/sql/qa/src/main/java/org/elasticsearch/xpack/sql/qa/geo/GeoDataLoader.java

```diff
@@ -73,25 +73,25 @@ private static String createOGCIndexRequest() throws Exception {
                 createString("name", createIndex);
```

Comment:
Since `geo_shape` support is still around, it would help to have both `geo_shape` and `shape` field types in the same dataset for both testing and example purposes. Unless I'm missing something it looks like all the geo_ fields were renamed.

Reply:
`geo_shape` and `shape` are both here to stay, they share pretty much most of the code paths, there is some test duplication between OGC tests and other tests, and `shape` implementation is much closer to OGC standard. So, I thought it just switching OGC test to `shape` would make sense and shouldn't reduce overall test coverage and we don't bloat the tests in the process.

Reply:
@costin does this makes sense, or should I make a copy of the tests for `shape`?

Reply:
@imotov A full copy is not needed however it would good to have a test that touches on a `shape` field (even if its just listing or doing a basic query) alongside a `geo_shape`.

Reply:
:+1: